src/tests/graphql/medicationRequest/graphql.medicationRequest.test.js

```diff
@@ -0,0 +1,109 @@
+                            return cb(err, resp);
+                        }),
+                (results, cb) =>
+                    request
+                        .post('/4_0_0/Patient/$merge')
+                        .send(patientBundleResource)
+                        .set(getHeaders())
+                        .expect(200, (err, resp) => {
+                            console.log('------- response 2 ------------');
+                            console.log(JSON.stringify(resp.body, null, 2));
+                            console.log('------- end response 2  ------------');
+                            return cb(err, resp);
+                        }),
+                (results, cb) =>
+                    request
+                        .post('/4_0_0/MedicationRequest/$merge')
+                        .send(medicationRequestBundleResource)
+                        .set(getHeaders())
+                        .expect(200, (err, resp) => {
+                            console.log('------- response 3 ------------');
+                            console.log(JSON.stringify(resp.body, null, 2));
+                            console.log('------- end response 3 ------------');
+                            return cb(err, resp);
+                        }),
+                (results, cb) =>
+                    request
+                        .post('/4_0_0/MedicationDispense/$merge')
+                        .send(medicationDispenseBundleResource)
+                        .set(getHeaders())
+                        .expect(200, (err, resp) => {
+                            console.log('------- response 4 ------------');
+                            console.log(JSON.stringify(resp.body, null, 2));
+                            console.log('------- end response 4 ------------');
+                            return cb(err, resp);
+                        }),
+                (results, cb) =>
+                    request
+                        .post('/graphqlv2')
+                        .send({
+                            'operationName': null,
+                            'variables': {},
+                            'query': graphqlQueryText
+                        })
+                        .set(getGraphQLHeaders())
+                        .expect(200, cb)
+                        .expect((resp) => {
+                            let body = resp.body;
+                            console.log('------- response graphql ------------');
+                            console.log(JSON.stringify(resp.body, null, 2));
+                            console.log('------- end response graphql  ------------');
+                            if (body.errors) {
+                                console.log(body.errors);
+                                expect(body.errors).toBeUndefined();
+                            }
+                            expect(body.data.medicationRequest.entry).toStrictEqual(expectedGraphQlResponse);
```

Comment:
FYI, We have a template for tests, might make this easier and more consistent going forward.
<img width="648" alt="image" src="https://user-images.githubusercontent.com/104457095/178066067-b7546949-a353-496f-b7f3-31a01eeb0763.png">